src/FormCheck.js

```diff
@@ -1,19 +1,26 @@
 import classNames from 'classnames';
-import React, { useContext, useMemo } from 'react';
 import PropTypes from 'prop-types';
-
-import { useBootstrapPrefix } from './ThemeProvider';
-import FormContext from './FormContext';
+import all from 'prop-types-extra/lib/all';
+import React, { useContext, useMemo } from 'react';
 import Feedback from './Feedback';
 import FormCheckInput from './FormCheckInput';
 import FormCheckLabel from './FormCheckLabel';
+import FormContext from './FormContext';
+import { useBootstrapPrefix } from './ThemeProvider';
 
 const propTypes = {
   /**
    * @default 'form-check'
    */
   bsPrefix: PropTypes.string,
 
+  /**
+   * A seperate bsPrefix used for custom controls
+   *
+   * @default 'custom-control'
+   */
+  bsCustomPrefix: PropTypes.string,
```

Comment:
How is this intended to be used? I don't see it getting hit at all.

Reply:
it's used below in the useBootstrap thing

Reply:
i mean i don't see this ever getting overridden or anything demonstrating this getting overridden

Reply:
O it's more an escape hatch for a user, same as bsPrefix, since we are using `custom-control` in the same way it seemed good to also expose it

Reply:
might be good to document/demo

Reply:
It documented i guess. I didn't want to make a big deal out of it and confuse normal users i guess!